### PR TITLE
fix: order of tabs, styling, and grammar in UC image creation guide

### PR DESCRIPTION
**Pull request by locnnil**
Address additional suggestion from #60: 
* change order of tabs to show UC24 first.
* improve sentences.

### CODE REVIEW

File: docs/how-to/uc-image-creation.rst [hunk 1]
```diff
@@ -391,4 +392,4 @@ The `Ubuntu Core documentation`_ is the best place to continue to learn about th
 .. _building Ubuntu Core images: https://ubuntu.com/core/docs/build-write-image
 .. _Ubuntu Core documentation: https://ubuntu.com/core/docs
 .. _flashing the image to a storage medium: https://ubuntu.com/core/docs/install-on-a-device
-.. _ubuntu-image assertion warning: https://forum.snapcraft.io/t/ubuntu-image-warning-kernel-snap/37774/3?u=farshidtz
\ No newline at end of file
+.. _ubuntu-image assertion warning: https://forum.snapcraft.io/t/ubuntu-image-warning-kernel-snap/37774/3?u=farshidtz
```
Comment (farshidtz) [inline docs/how-to/uc-image-creation.rst:395]:
```suggestion
.. _ubuntu-image assertion warning: https://forum.snapcraft.io/t/ubuntu-image-warning-kernel-snap/37774/3
```

File: docs/how-to/uc-image-creation.rst [hunk 2]
```diff
@@ -47,6 +41,13 @@ Below are example model assertions, describing the Ubuntu Core image content for
         The ``console-conf`` snap added here is only to allow interactive user and network configuration.
```
Comment (farshidtz) [inline docs/how-to/uc-image-creation.rst:41]: Could you also include the object defining console-conf snap to `emphasize-lines`? It should be highlighted as that is a change compared to the original model assertion.

**Review by farshidtz — CHANGES_REQUESTED** [review]
There are still two more tab groups containing UC24/UC22 tabs with the opposite order.

File: docs/how-to/uc-image-creation.rst (hunk 2 above)
Comment (locnnil) [reply docs/how-to/uc-image-creation.rst:41]: Done!

**Review by farshidtz — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**